components/series/utils.ts

```diff
@@ -61,18 +61,18 @@ export const nftFn = (a: any): RowSeries => {
   }
 }
 
-const tokenDecimals = store.getters[
-  'chain/getChainPropertiesTokenDecimals'
-] as any
-const formatNumber = (val: SimpleSeriesNFT) =>
-  Number(formatBalance(val.price, tokenDecimals, false, true))
-const sumFn = (acc: number, val: SimpleSeriesNFT) => acc + formatNumber(val)
+// const tokenDecimals = store.getters[
+//   'chain/getChainPropertiesTokenDecimals'
+// ] as any
+// const formatNumber = (val: SimpleSeriesNFT) =>
+//   Number(formatBalance(val.price, tokenDecimals, false, true))
+// const sumFn = (acc: number, val: SimpleSeriesNFT) => acc + formatNumber(val)
 const soldFn = (acc: number, val: SimpleSeriesNFT) =>
   val.issuer !== val.currentOwner ? acc + 1 : acc
 const uniqueFn = (acc: Set<string>, val: SimpleSeriesNFT) =>
   acc.add(val.metadata)
-const onlyOwned = ({ issuer, currentOwner }: SimpleSeriesNFT) =>
-  issuer === currentOwner
+// const onlyOwned = ({ issuer, currentOwner }: SimpleSeriesNFT) =>
+//   issuer === currentOwner
 // const onlyBuyEvents = ({ events }: SimpleSeriesNFT) => events.filter((e: { interaction: string }) => e.interaction === 'BUY')
```

Comment:
Commented out since they were not used (averagePrice is not returned)

Reply:
👀